CMakeLists.txt

```diff
@@ -465,14 +468,17 @@ else()
   set(FOLLY_BENCHMARK Folly::follybenchmark)
 endif()
 
-# DWIO (ORC/DWRF) and Substrait depend on protobuf.
-if(${VELOX_BUILD_MINIMAL_WITH_DWIO}
-   OR ${VELOX_ENABLE_HIVE_CONNECTOR}
-   OR ${VELOX_ENABLE_SUBSTRAIT})
-  # Locate or build protobuf.
-  set_source(Protobuf)
-  resolve_dependency(Protobuf 3.21 EXACT)
-  include_directories(${Protobuf_INCLUDE_DIRS})
+if(VELOX_ENABLE_GCS)
+  set_source(absl)
+  resolve_dependency(absl 20240116 EXACT)
+
+  set_source(gRPC)
+  resolve_dependency(gRPC 1.48.1 EXACT)
+
```

Comment:
You can move these into the gcp.cmake, they don't have to be called in the main cmake, that way it will only build them if gcp isn't found on the system (when it's found on the system all deps are also there).

Reply:
Yes, I agree.